apps/serialization/messages.proto

```diff
@@ -223,27 +227,28 @@ message Effect {
 }
 
 message Item {
-  string name = 2;
+  optional string name = 2;
 }
 
 message Projectile {
-  uint64 damage = 1;
-  uint64 owner_id = 2;
+  optional uint64 damage = 1;
+  optional uint64 owner_id = 2;
   ProjectileStatus status = 3;
-  string skill_key = 4;
+  optional string skill_key = 4;
 }
 
 enum ProjectileStatus {
-  ACTIVE = 0;
-  EXPLODED = 1;
-  CONSUMED = 2;
+  PROJECTILE_STATUS_UNDEFINED = 0;
```

Comment:
This new enum value `PROJECTILE_STATUS_UNDEFINED` and the similar one in `CrateStatus` is a [protobuf best practice](https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum) cause it been the 0 value will serve as the default for the enum. You are not supposed to actually have business logic tied to it, for example I only use it in the client as a null check

Additionally you probably noticed that its called `PROJECTILE_STATUS_UNDEFINED` rather than `UNDEFINED` that's cause all enum names share a namespace and the protobuf C# compiler didn't like it